docs/scratch_orgs.rst

```diff
@@ -1,38 +1,253 @@
+
+.. important :: Be sure to use the ``--setdefaultdevhubusername`` option!
+
+If you have the ``sfdx`` command installed, are connected to your devhub, and set the ``defaultdevhubusername`` config setting (use the ``sfdx force:config:list`` command to verify!), you're now ready to use ``cci`` with ``sfdx``.
+
+.. note:: SFDX supports multiple DevHubs, so CumulusCI uses the one set as ``defaultdevhubusername`` when creating scratch orgs.
+
+Learn more about Salesforce DX at https://developer.salesforce.com/platform/dx.
+
+
+
+Predefined Orgs
 ---------------
 
-Create a scratch org
+CumulusCI comes with predefined org configurations. Every org's keychain starts with these configurations ready and available to be turned into a live scratch org. To see the predefined org configuations in your project:
+
+.. code-block:: console
+
+    $ cci org list
+
+
++-------------+--------------------------+-----------------------+----------+
+|   Org       | Role                     | Configuration File    | Lifespan |
++=============+==========================+=======================+==========+
+| ``dev``     | Development workflows    | ``orgs/dev.json``     | 7 days   |
++-------------+--------------------------+-----------------------+----------+
+| ``qa``      | Testing workflows        | ``orgs/dev.json``     | 7 days   |
++-------------+--------------------------+-----------------------+----------+
+| ``feature`` | Continuous Integration   | ``orgs/dev.json``     | 1 day    |
++-------------+--------------------------+-----------------------+----------+
+| ``beta``    | | Continuous Integration | ``orgs/beta.json``    | 1 day    |
+|             | | Hands-On Testing       |                       |          |
++-------------+--------------------------+-----------------------+----------+
+| ``release`` | | Continuous Integration | ``orgs/release.json`` | 1 day    |
+|             | | Hands-On Testing       |                       |          |
+|             | | Demo Workflow          |                       |          |
++-------------+--------------------------+-----------------------+----------+
+
+* ``dev`` is for development workflows and uses the ``orgs/dev.json`` configuration file. It has a seven-day lifespan.
+* ``qa`` is for testing workflows. ``qa`` and ``dev`` are the same out of the box, but can be customized to suit the needs of the project. It has a seven-day lifespan.
+* ``feature`` is for continuous integration and uses the ``orgs/dev.json`` configuration file. It has a one-day lifespan.
+* ``beta`` is for continuous integration or hands-on testing. It uses the ``orgs/beta.json`` configuration. It has a one-day lifespan.
+* ``release`` is for continuous integration, hands-on testing, or demo workflow. It uses the ``orgs/release.json`` configuration. It has a one-day lifespan.
+
+If your project has customized the available orgs, you'll see more orgs, or differences in configuration, compared to a new project.
```

Comment:
This kinda goes without saying. Unless I'm missing an insight here, I recommend scrapping it.

Reply:
Maybe it can be made clearer/

```suggestion
If your project has customized org configurations, your listing may include more configurations than shown above, and your project's versions of the standard configurations may be different.
```

Reply:
I think it's worth keeping. It reconfirms to newer users that the output will likely be different if they're beginning work on an existing project that has been highly configured.